client/src/components/WorkspacePanels/MainWorkspacePanel/MainWorkspace.js

```diff
@@ -32,9 +39,26 @@ const MainWorkspace = ({ music }) => {
                 <TrackObject
                   musicProps={props}
                   index={index}
-                  row={i}
-                  key={"trackobj-" + i + "-" + props.id}
-                />
+                  key={"trackobject-" + id}
+                >
+                  {(provided) => (
+                    <div
+                      className="track-object"
+                      ref={provided.innerRef}
+                      {...provided.draggableProps}
+                      {...provided.dragHandleProps}
+                    >
+                      <button
+                        className="x-button"
+                        onClick={(event) => handleDelete(event, i, index)}
+                      >
+                        X
+                      </button>{" "}
+                      <div className="type-display">{type}</div>
+                      <div className="track-data">{name}</div>
+                    </div>
+                  )}
+                </Draggable>
```

Comment:
I believe on master, this part of the code does not exist (I made it modular, so it should be in the `TrackObject.js` file. Could you please pull master and merge the code?